### PR TITLE
updating package names from r1soft to serverbackup to fix issue #3

### PR DESCRIPTION
**Pull request by anthonysomerset**
as per issue #3 here is a pull request that changes the package names and also bumps the version number for people using the module management systems in puppet

### CODE REVIEW

File: manifests/packages.pp [hunk 1]
```diff
@@ -19,14 +19,14 @@
...
             ensure  => installed,
-            require => Class['r1soft_cdp_agent::repo'],
-            before => Package['r1soft-cdp-enterprise-agent'],
+            require => Class['serverbackup_cdp_agent::repo'],
+            before => Package['serverbackup-cdp-enterprise-agent'],
         }
     }
 
-    package { 'r1soft-cdp-enterprise-agent':
+    package { 'serverbackup-cdp-enterprise-agent':
```
Comment (lboynton) [inline manifests/packages.pp]: Looks like the package has been renamed to 'serverbackup-enterprise-agent'.

**Comment by lboynton** [comment]
I notice you have changed a lot of the single quotes from `'` to `‘`, have you been using a word processor instead of a text editor?

**Comment by anthonysomerset** [comment]
ah my bad - autocorrect in textedit for mac even in plaintext mode

**Comment by anthonysomerset** [comment]
both should now be fixed

**Comment by anthonysomerset** [comment]
fixed - note to self - must stop using textedit for quick edits :)

**Comment by lboynton** [comment]
Merged, thanks @anthonysomerset!

**Pull request merged**